app/models/comment.rb

```diff
@@ -1,9 +1,9 @@
 class Comment < ApplicationRecord
-  # belongs_to :user, foreign_key: 'author_id'
-  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :user, foreign_key: 'author_id'
+  # belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Great work on making the associations well 👍 but I guess you made this line commented 😨 Kindly remove the commented piece of code to make your codebase clean to follow Ruby best practices 😊

Reply:
fixed